recipes/_r-mutex/LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+BSD 3-clause license
+Copyright (c) 2015-2019, conda-forge
```

Comment:
This recipe originates from Anaconda Inc., this should be reflected here.

Reply:
Thanks for spotting it! Fixed.